From 00be01b82bbd64a4a08956b050529491941e6460 Mon Sep 17 00:00:00 2001
From: "yi.zhang1" <yi.zhang1@amlogic.com>
Date: Thu, 19 Oct 2023 16:46:14 +0800
Subject: [PATCH 05/14] amlbian: t7c hdmi only support isp [1/1]

PD#SWPL-142978

Problem:
t7c hdmi only isp not OK.

Solution:
add new device-tree compatible.

Verify:
local

Signed-off-by: yi.zhang1 <yi.zhang1@amlogic.com>
Change-Id: I572090b16b51e5939c4e0c4748011304b7df56f3
---
 recipes-kernel/isp/files/isp_prepare.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/recipes-kernel/isp/files/isp_prepare.sh b/recipes-kernel/isp/files/isp_prepare.sh
index 508ec2c1..7f90ebc6 100755
--- a/recipes-kernel/isp/files/isp_prepare.sh
+++ b/recipes-kernel/isp/files/isp_prepare.sh
@@ -4,6 +4,8 @@
 CHIP_NAME=""
 if cat /proc/device-tree/compatible | grep "t7c_a311d2_an400-4g"; then
     CHIP_NAME="t7c"
+elif cat /proc/device-tree/compatible | grep "t7c_a311d2_an400-hdmitx-only"; then
+    CHIP_NAME="t7c"
 elif cat /proc/device-tree/compatible | grep "t7_a311d2_an400"; then
     CHIP_NAME="t7"
 elif cat /proc/device-tree/compatible | grep "g12b_w400_a"; then
-- 
2.25.1

